client/packages/requisitions/src/RequestRequisition/DetailView/ContentArea.tsx

```diff
@@ -1,22 +1,61 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import {
+  AppSxProp,
   DataTable,
   NothingHere,
+  useRowStyle,
   useTranslation,
 } from '@openmsupply-client/common';
 import { RequestLineFragment, useHideOverStocked, useRequest } from '../api';
+import { isRequestLinePlaceholderRow } from '../../utils';
+import { RequestItem } from '../../types';
 
 interface ContentAreaProps {
   onAddItem: () => void;
   onRowClick: null | ((line: RequestLineFragment) => void);
 }
 
+const useHighlightPlaceholderRows = (
+  rows: RequestLineFragment[] | RequestItem[] | undefined
+) => {
+  const { setRowStyles } = useRowStyle();
+
+  useEffect(() => {
+    if (!rows) return;
+    const placeholders = [];
+
+    for (const row of rows) {
+      if ('requestedQuantity' in row) {
+        if (isRequestLinePlaceholderRow(row)) {
+          placeholders.push(row.id);
+        }
+      } else {
+        const hasPlaceholder = row.lines.some(isRequestLinePlaceholderRow);
+        if (hasPlaceholder) {
+          row.lines.forEach(line => {
+            if (isRequestLinePlaceholderRow(line)) {
+              placeholders.push(line.id);
+            }
+          });
+          placeholders.push(row.id);
+        }
+      }
+    }
```

Comment:
we don't have the 'group by item' for internal orders, so this can be simplified:
```suggestion
  useEffect(() => {
    if (!rows) return;

    const placeholders = rows
      .filter(isRequestLinePlaceholderRow)
      .map(row => row.id);
    const style: AppSxProp = {
      color: theme => theme.palette.secondary.light,
    };
    setRowStyles(placeholders, style);
  }, [rows, setRowStyles]);
```